### PR TITLE
Grd 75933

### PR DESCRIPTION
**Pull request by RefaelAdi**
- All the offline plug-ins are added to the release v.1.5.0:
  https://github.com/IBM/universal-connectors/releases/tag/v1.5.0 
- The Download URLs to the plug-in are updated accordingly in every plug-in's readme file. 
- Broken URLs are fixed
- In case there is no download URL, I added back the offline plug-in till the readme is updated again (for example: S3)
- **PLEASE DO NOT MERGE**

### CODE REVIEW

File: filter-plugin/logstash-filter-pubsub-apachesolr-guardium/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The plug-in is free and open-source (Apache 2.0). It can be used as a starting p
 
 ### Prerequisites
 
-Download the [Logstash Offline package](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-apachesolr-guardium/PubSubApacheSolrPackage/guardium_logstash-offline-plugin-pubsub-apache-solr-gcp.zip) for the Google Cloud Apache Solr PubSub filter plugin, and upload it to the Guardium Data Protection collector.
+Download the [Logstash Offline package](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-apache_solr_gcp_connector.zip) for the Google Cloud Apache Solr PubSub filter plugin, and upload it to the Guardium Data Protection collector.
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-pubsub-apachesolr-guardium/README.md:21]: pubsub plugins won't work with the Logstash8.x build. Need to point them to links

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 2]
```diff
@@ -102,12 +102,12 @@ The Guardium universal connector is the Guardium entry point for native audit/da
 ### Before you begin
 * Configure the policies you require. See [policies](/docs/#policies) for more information.
 * You must have permission for the S-Tap Management role. The admin user includes this role by default
-* Download the [guardium_logstash-offline-plugins-ps-bigQuery.zip](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-bigquery-guardium/BigQueryOverPubSubPackage/guardium_logstash-offline-plugins-ps-bigQuery.zip) plug-in. (Do not unzip the offline-package file throughout the procedure). This step is not necessary for Guardium Data Protection v12.0 and later.
+* Download the [logstash-filter-big_query_guardium_filter.zip](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-big_query_guardium_filter.zip) plug-in. (Do not unzip the offline-package file throughout the procedure). This step is not necessary for Guardium Data Protection v12.0 and later.
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:105]: pubsub..

File: filter-plugin/logstash-filter-pubsub-firebase-realtime-guardium/README.md [hunk 3]
```diff
@@ -137,12 +137,12 @@ The Guardium universal connector is the Guardium entry point for native audit/da
 ### Before you begin
 *  Configure the policies you require. See [policies](/../../#policies) for more information.
 * You must have permission for the S-Tap Management role. The admin user includes this role by default. 
-* Download the [guardium_logstash-offline-plugin-Firebase.zip](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-firebase-realtime-guardium/PubSubFirebasePackage/guardium_logstash-offline-plugin-Firebase.zip) plug-in. (Do not unzip the offline-package file throughout the procedure). This step is not necessary for Guardium Data Protection v12.0 and later.
+* Download the [logstash-filter-fire_base_guardium_filter.zip](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-fire_base_guardium_filter.zip) plug-in. (Do not unzip the offline-package file throughout the procedure). This step is not necessary for Guardium Data Protection v12.0 and later.
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-pubsub-firebase-realtime-guardium/README.md:140]: pubsub

File: filter-plugin/logstash-filter-pubsub-firestore-guardium/README.md [hunk 4]
```diff
@@ -119,12 +119,12 @@ The Guardium universal connector is the Guardium entry point for native audit/da
    * Configure the policies you require. See [policies](/docs/#policies) for more information.
    * You must have permission for the S-Tap Management role. The admin user includes this role by default.
 
-   * Download the [guardium_logstash-offline-plugins-ps-firestore.zip](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-firestore-guardium/PubSubFireStorePackage/guardium_logstash-offline-plugins-ps-firestore.zip) plug-in. (Do not unzip the offline-package file throughout the procedure). This step is not necessary for Guardium Data Protection v12.0 and later.
+   * Download the [logstash-filter-fire_store_guardium_filter.zip](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-fire_store_guardium_filter.zip) plug-in. (Do not unzip the offline-package file throughout the procedure). This step is not necessary for Guardium Data Protection v12.0 and later.
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-pubsub-firestore-guardium/README.md:122]: pubsub..

File: filter-plugin/logstash-filter-pubsub-mysql-guardium/README.md [hunk 5]
```diff
@@ -25,8 +25,8 @@ The plug-in is free and open-source (Apache 2.0). It can be used as a starting p
 
 ### Prerequisites
 Download the relevant plugin based on the version of the Guardium.
-    1. For the Guardium 11.x, download [logstash-filter-pubsub-mysql-guardium-7.16.3.zip](PubSubMySQLPackage/logstash-filter-pubsub-mysql-guardium-7.16.3.zip).
-    2. For the Guardium 12.x, download [logstash-filter-pubsub-mysql-guardium-8.3.3.zip](PubSubMySQLPackage/logstash-filter-pubsub-mysql-guardium-8.3.3.zip)
+    1. For the Guardium 11.x, download [logstash-filter-pubsub-mysql-guardium-7.16.3.zip](https://github.com/IBM/universal-connectors/raw/main/filter-plugin/logstash-filter-pubsub-mysql-guardium/PubSubMySQLPackage/logstash-filter-pubsub-mysql-guardium-7.16.3.zip).
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-pubsub-mysql-guardium/README.md:28]: pubsub- good example of how it should look like

File: filter-plugin/logstash-filter-pubsub-postgresql-guardium/README.md [hunk 6]
```diff
@@ -17,7 +17,7 @@ The plug-in is free and open-source (Apache 2.0). It can be used as a starting p
 ## Documentation
 
 ### Prerequisites
-Download the [Logstash Offline package](../../filter-plugin/logstash-filter-pubsub-postgresql-guardium/PubSubPostgreSQLPackage/logstash-offline-plugins-filter-pubsub-postgresql-guardium.zip) that includes both the Logstash Google PubSub input plugin and the PostgreSQL PubSub filter plugin, and upload it to the gmachine.
+Download the [Logstash Offline package](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-pubsub-postgresql-guardium.zip) that includes both the Logstash Google PubSub input plugin and the PostgreSQL PubSub filter plugin, and upload it to the gmachine.
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-pubsub-postgresql-guardium/README.md:20]: pubsub..

File: filter-plugin/logstash-filter-pubsub-spanner-guardium/README.md [hunk 7]
```diff
@@ -121,12 +121,12 @@ The Guardium universal connector is the Guardium entry point for native audit/da
 * Configure the policies you require. See [policies](/docs/#policies) for more information.
 * You must have permission for the S-Tap Management role. The admin user includes this role by default.
 
-* Download the [guardium_logstash-offline-plugins-gcp-pubsub-spanner.zip](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-spanner-guardium/SpannerOverPubSubPackage/guardium_logstash-offline-plugins-gcp-pubsub-spanner.zip) plug-in. (Do not unzip the offline-package file throughout the procedure). This step is not necessary for Guardium Data Protection v12.0 and later.
+* Download the [logstash-filter-spanner_db_guardium_filter.zip](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-spanner_db_guardium_filter.zip) plug-in. (Do not unzip the offline-package file throughout the procedure). This step is not necessary for Guardium Data Protection v12.0 and later.
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-pubsub-spanner-guardium/README.md:124]: pubsub..

**Review by ofer-haim — COMMENTED** [review]
1. see pubsub comments.. mysql looks ok 
2. hdfs, percona- why did we need to add these zips again? is it in purpose?

File: filter-plugin/logstash-filter-pubsub-apachesolr-guardium/README.md (hunk 1 above)
Comment (RefaelAdi) [reply filter-plugin/logstash-filter-pubsub-apachesolr-guardium/README.md]: updated for all pubsub plug-ins

**Review by apurva-birajdar — APPROVED** [review]

**Review by chirag-ibm — APPROVED** [review]

File: filter-plugin/logstash-filter-pubsub-apachesolr-guardium/README.md [hunk 8]
```diff
@@ -156,12 +150,15 @@ The Guardium universal connector is the Guardium entry point for native audit/da
 ## Before you begin
 *  Configure the policies you require. See [policies](/docs/#policies) for more information.
 * You must have permission for the S-Tap Management role. The admin user includes this role by default.
-* Download the [guardium_logstash-offline-plugin-pubsub-apache-solr-gcp.zip](PubSubApacheSolrPackage/guardium_logstash-offline-plugin-pubsub-apache-solr-gcp.zip) plug-in. (Do not unzip the offline-package file throughout the procedure). This step is not necessary for Guardium Data Protection v12.0 and later.
+* Download the relevant plugin based on the version of the Guardium.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-apachesolr-guardium/README.md:153]: plug-in for the Guardium version that you are using.

File: filter-plugin/logstash-filter-pubsub-apachesolr-guardium/README.md [hunk 9]
```diff
@@ -156,12 +150,15 @@ The Guardium universal connector is the Guardium entry point for native audit/da
...
+1. For the Guardium 11.x, download the [Logstash_Offline_package_7.x](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-apachesolr-guardium/PubSubApacheSolrPackage/guardium_logstash-offline-plugin-pubsub-apache-solr-gcp.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-apachesolr-guardium/README.md:154]: For Guardium 11.x, download the ... file.

File: filter-plugin/logstash-filter-pubsub-apachesolr-guardium/README.md [hunk 10]
```diff
@@ -156,12 +150,15 @@ The Guardium universal connector is the Guardium entry point for native audit/da
...
+2. For the Guardium 12.x, download the [Logstash_Offline_package_8.x](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-apache_solr_gcp_connector.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-apachesolr-guardium/README.md:155]: For Guardium 12.x, download the...file.

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 11]
```diff
@@ -102,7 +102,9 @@ The Guardium universal connector is the Guardium entry point for native audit/da
 ### Before you begin
 * Configure the policies you require. See [policies](/docs/#policies) for more information.
 * You must have permission for the S-Tap Management role. The admin user includes this role by default
-* Download the [guardium_logstash-offline-plugins-ps-bigQuery.zip](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-bigquery-guardium/BigQueryOverPubSubPackage/guardium_logstash-offline-plugins-ps-bigQuery.zip) plug-in. (Do not unzip the offline-package file throughout the procedure). This step is not necessary for Guardium Data Protection v12.0 and later.
+* Download the relevant plugin based on the version of the Guardium.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:105]: plug-in for the Guardium version that you are using.

File: filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md [hunk 12]
```diff
@@ -102,7 +102,9 @@ The Guardium universal connector is the Guardium entry point for native audit/da
...
+1. For the Guardium 11.x, download the [Logstash_Offline_package_7.x](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-bigquery-guardium/BigQueryOverPubSubPackage/guardium_logstash-offline-plugins-ps-bigQuery.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-bigquery-guardium/README.md:106]: For Guardium 11.x, download the ... file.

File: filter-plugin/logstash-filter-pubsub-firebase-realtime-guardium/README.md [hunk 13]
```diff
@@ -136,8 +136,10 @@ The Guardium universal connector is the Guardium entry point for native audit/da
 
 ### Before you begin
 *  Configure the policies you require. See [policies](/../../#policies) for more information.
-* You must have permission for the S-Tap Management role. The admin user includes this role by default. 
-* Download the [guardium_logstash-offline-plugin-Firebase.zip](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-firebase-realtime-guardium/PubSubFirebasePackage/guardium_logstash-offline-plugin-Firebase.zip) plug-in. (Do not unzip the offline-package file throughout the procedure). This step is not necessary for Guardium Data Protection v12.0 and later.
+* You must have permission for the S-Tap Management role. The admin user includes this role by default.
+* Download the relevant plugin based on the version of the Guardium.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-firebase-realtime-guardium/README.md:140]: plug-in for the Guardium version that you are using.

File: filter-plugin/logstash-filter-pubsub-firebase-realtime-guardium/README.md [hunk 14]
```diff
@@ -136,8 +136,10 @@ The Guardium universal connector is the Guardium entry point for native audit/da
...
+1. For the Guardium 11.x, download the [Logstash_Offline_package_7.x](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-firebase-realtime-guardium/PubSubFirebasePackage/guardium_logstash-offline-plugin-Firebase.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-firebase-realtime-guardium/README.md:141]: For Guardium 11.x, download the ... file.

File: filter-plugin/logstash-filter-pubsub-firebase-realtime-guardium/README.md [hunk 15]
```diff
@@ -136,8 +136,10 @@ The Guardium universal connector is the Guardium entry point for native audit/da
...
+2. For the Guardium 12.x, download the [Logstash_Offline_package_8.x](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-fire_base_guardium_filter.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-firebase-realtime-guardium/README.md:142]: For Guardium 12.x, download the ... file.

File: filter-plugin/logstash-filter-pubsub-firestore-guardium/README.md [hunk 16]
```diff
@@ -118,8 +118,9 @@ The Guardium universal connector is the Guardium entry point for native audit/da
 ### Before you begin
    * Configure the policies you require. See [policies](/docs/#policies) for more information.
    * You must have permission for the S-Tap Management role. The admin user includes this role by default.
-
-   * Download the [guardium_logstash-offline-plugins-ps-firestore.zip](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-firestore-guardium/PubSubFireStorePackage/guardium_logstash-offline-plugins-ps-firestore.zip) plug-in. (Do not unzip the offline-package file throughout the procedure). This step is not necessary for Guardium Data Protection v12.0 and later.
+   * Download the relevant plugin based on the version of the Guardium.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-firestore-guardium/README.md:121]: plug-in for the Guardium version that you are using.

File: filter-plugin/logstash-filter-pubsub-firestore-guardium/README.md [hunk 17]
```diff
@@ -118,8 +118,9 @@ The Guardium universal connector is the Guardium entry point for native audit/da
...
+1. For the Guardium 11.x, download the [Logstash_Offline_package_7.x](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-firestore-guardium/PubSubFireStorePackage/guardium_logstash-offline-plugins-ps-firestore.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-firestore-guardium/README.md:122]: For Guardium 11.x, download the ... file.

File: filter-plugin/logstash-filter-pubsub-firestore-guardium/README.md [hunk 18]
```diff
@@ -118,8 +118,9 @@ The Guardium universal connector is the Guardium entry point for native audit/da
...
+2. For the Guardium 12.x, download the [Logstash_Offline_package_8.x](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-fire_base_guardium_filter.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-firestore-guardium/README.md:123]: For Guardium 12.x, download the ... file.

File: filter-plugin/logstash-filter-pubsub-mysql-guardium/README.md [hunk 19]
```diff
@@ -134,8 +127,8 @@ logName="projects/<PROJECT_ID>/logs/cloudsql.googleapis.com%2Fmysql.err")
 
 ## Installation
 To install this plug-in, you need to download the relevant plugin based on the version of the Guardium.
-1. For the Guardium 11.x, download [logstash-filter-pubsub-mysql-guardium-7.16.3.zip](PubSubMySQLPackage/logstash-filter-pubsub-mysql-guardium-7.16.3.zip).
-2. For the Guardium 12.x, download [logstash-filter-pubsub-mysql-guardium-8.3.3.zip](PubSubMySQLPackage/logstash-filter-pubsub-mysql-guardium-8.3.3.zip)
+1. For the Guardium 11.x, download [logstash-filter-pubsub-mysql-guardium-7.16.3.zip](https://github.com/IBM/universal-connectors/raw/main/filter-plugin/logstash-filter-pubsub-mysql-guardium/PubSubMySQLPackage/logstash-filter-pubsub-mysql-guardium-7.16.3.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-mysql-guardium/README.md:130]: For Guardium 11.x, download the ... file.

File: filter-plugin/logstash-filter-pubsub-mysql-guardium/README.md [hunk 20]
```diff
@@ -134,8 +127,8 @@ logName="projects/<PROJECT_ID>/logs/cloudsql.googleapis.com%2Fmysql.err")
...
+2. For the Guardium 12.x, download [logstash-filter-pubsub-mysql-guardium-8.3.3.zip](https://github.com/IBM/universal-connectors/raw/main/filter-plugin/logstash-filter-pubsub-mysql-guardium/PubSubMySQLPackage/logstash-filter-pubsub-mysql-guardium-8.3.3.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-mysql-guardium/README.md:131]: For Guardium 12.x, download the ... file.

File: filter-plugin/logstash-filter-pubsub-postgresql-guardium/README.md [hunk 21]
```diff
@@ -55,7 +53,9 @@ Please refer to the [input plugin's repository](../../input-plugin/logstash-inpu
 - `clientIP` and `serverIP` fields are populated with `0.0.0.0`, as this information is not embedded in the messages pulled from Google Cloud.
 
 ## Installation
-To install this plug-in, you need to download the [offline pack](PubSubPostgreSQLPackage/logstash-offline-plugins-filter-pubsub-postgresql-guardium.zip) and upload the file to the gmachine
+To install this plug-in, you need to download the relevant plugin based on the version of the Guardium.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-postgresql-guardium/README.md:56]: plug-in for the Guardium version that you are using.

File: filter-plugin/logstash-filter-pubsub-postgresql-guardium/README.md [hunk 22]
```diff
@@ -55,7 +53,9 @@ Please refer to the [input plugin's repository](../../input-plugin/logstash-inpu
...
+1. For the Guardium 11.x, download the [Logstash_Offline_package_7.x](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-postgresql-guardium/PubSubPostgreSQLPackage/logstash-offline-plugins-filter-pubsub-postgresql-guardium.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-postgresql-guardium/README.md:57]: For Guardium 11.x, download the ... file.

File: filter-plugin/logstash-filter-pubsub-postgresql-guardium/README.md [hunk 23]
```diff
@@ -55,7 +53,9 @@ Please refer to the [input plugin's repository](../../input-plugin/logstash-inpu
...
+2. For the Guardium 12.x, download the [Logstash_Offline_package_8.x](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-pubsub-postgresql-guardium.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-postgresql-guardium/README.md:58]: For Guardium 12.x, download the ... file.

File: filter-plugin/logstash-filter-pubsub-spanner-guardium/README.md [hunk 24]
```diff
@@ -120,8 +120,11 @@ The Guardium universal connector is the Guardium entry point for native audit/da
 ### Before you begin
 * Configure the policies you require. See [policies](/docs/#policies) for more information.
 * You must have permission for the S-Tap Management role. The admin user includes this role by default.
+* Download the relevant plugin based on the version of the Guardium.
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-spanner-guardium/README.md:123]: plug-in for the Guardium version that you are using.

File: filter-plugin/logstash-filter-pubsub-spanner-guardium/README.md [hunk 25]
```diff
@@ -120,8 +120,11 @@ The Guardium universal connector is the Guardium entry point for native audit/da
...
+1. For the Guardium 11.x, download the [Logstash_Offline_package_7.x](https://github.com/IBM/universal-connectors/raw/release-v1.2.0/filter-plugin/logstash-filter-pubsub-spanner-guardium/SpannerOverPubSubPackage/guardium_logstash-offline-plugins-gcp-pubsub-spanner.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-spanner-guardium/README.md:124]: For Guardium 11.x, download the ... file.

File: filter-plugin/logstash-filter-pubsub-spanner-guardium/README.md [hunk 26]
```diff
@@ -120,8 +120,11 @@ The Guardium universal connector is the Guardium entry point for native audit/da
...
+2. For the Guardium 12.x, download the [Logstash_Offline_package_8.x](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-spanner_db_guardium_filter.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-pubsub-spanner-guardium/README.md:125]: For Guardium 12.x, download the ... file.

File: filter-plugin/logstash-filter-snowflake-guardium/README.md [hunk 27]
```diff
@@ -82,7 +82,7 @@ you can turn it off. To do this, set the value to `false` for the following two
 ### Before you begin
 1. Configure the policies you require. See [policies](/docs/#policies) for more information. 
 2. You must have permission for the S-Tap Management role. The admin user includes this role by default.
-3. Download the [logstash-filter-guardium_snowflake_filter-1.0.0.zip](SnowflakeOverJbdcPackage/Snowflake/logstash-offline-plugins-7.12.1.zip)
+3. Download the [logstash-filter-guardium_snowflake_filter.zip](https://github.com/IBM/universal-connectors/releases/download/v1.5.0/logstash-filter-guardium_snowflake_filter.zip)
```
Comment (AlexaNahum) [inline filter-plugin/logstash-filter-snowflake-guardium/README.md:85]: Download the ... file.

**Pull request merged**